test/functorch/test_control_flow.py

```diff
@@ -1174,6 +1203,225 @@ def fwbw(map_op, f, x, y):
         fake_outs = fwbw(_fake_map, f, x, y)
         self.assertEqual(true_outs, fake_outs)
 
+    @unittest.skipIf(not torch.cuda.is_available(), "Test requires CUDA.")
+    def test_generic_associative_scan_simple(self):
```

Comment:
Nit: change test name to test_pointwise_associative_scan_reverse_simple?

Reply:
Done